README.md

```diff
@@ -1552,6 +1552,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rohan Hari](https://github.com/rohan-hari)
   - [Rohan Hazari](https://github.com/Rohan-Hazari)
   - [Rohan Nair](https://github.com/rohan9454)
+  - [Rohan singh](https://github.com/singh1475)
```

Comment:
```suggestion
  - [Rohan Singh](https://github.com/singh1475)
```

Hi! Thanks for your contribution 🚀 Kindly ask you to add the suggestion to use a capital letters for first name and last name 🙏